mllib/src/main/scala/org/apache/spark/mllib/api/python/PythonMLLibAPI.scala

```diff
@@ -1093,6 +1093,27 @@ private[python] class PythonMLLibAPI extends Serializable {
     LinearDataGenerator.generateLinearRDD(
       sc, nexamples, nfeatures, eps, nparts, intercept)
   }
+
+  /**
+   * Wrapper around Statistics.kolmogorovSmirnovTestWrapper with default params.
+   */
+  def kolmogorovSmirnovTestWrapper(
+      data: JavaRDD[Double],
+      distName: String): KolmogorovSmirnovTestResult = {
+    Statistics.kolmogorovSmirnovTest(data, distName)
+  }
+
+  /**
+   * Wrapper around Statistics.kolmogorovSmirnovTestWrapper.
+   */
+  def kolmogorovSmirnovTestWrapper(
+      data: JavaRDD[Double],
+      distName: String,
+      params: JList[Double]): KolmogorovSmirnovTestResult = {
+    val seqParams = params.asScala.toSeq
+    Statistics.kolmogorovSmirnovTest(data, distName, seqParams: _*)
+  }
+
```

Comment:
I'll defer to @mengxr and @sryza but just wondering: do we need the 2 separate wrappers? seems to me that 1 should do (the last one). If params is passed in empty on the python side, that should just be an empty list, and the asScala.toSeq, will make it an empty sequence.